BioCro/DARPA/upload_params_data/BETY/Compare_BETY_to_upload.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Compare BETY and upload_params_data"
+---
+  
+  This script compares the results of what is in BETY to what has been uploaded to it, in an effort to identify mistakes or duplicates. 
+
+Data from BETY is selecte with the following SQL query: select * from traits where treatment_id in (9000000015, 9000000016, 9000000017, 9000000018, 9000000019, 9000000020)
+The results are saved as csv in the upload_params_data/BETY folder as the most recent 'update_at' date. 
+
+Load libraries and both sets of data
+```{r}
+options(scipen = 999)
+library(dplyr)
+library(arsenal)
+#import newest BETY file
```

Comment:
Usually when I feel compelled to add comments to code chunks, that's a sign that the code associated with each comment should probably be its own chunk. I'd recommend splitting this code chunk up into three chunks with corresponding text descriptions.

Reply:
And I think it's a common convention for the loading in of libraries to be its own code chunk anyways.